tests/garage/sampler/test_multiprocessing_sampler.py

```diff
@@ -67,16 +66,16 @@ def test_update_envs_env_update():
                             n_workers=n_workers)
     sampler = MultiprocessingSampler.from_worker_factory(workers, policy, env)
     episodes = sampler.obtain_samples(0,
-                                      161,
+                                      500,
                                       np.asarray(policy.get_param_values()),
                                       env_update=tasks.sample(n_workers))
     mean_rewards = []
     goals = []
     for eps in episodes.split():
         mean_rewards.append(eps.rewards.mean())
         goals.append(eps.env_infos['task'][0]['goal'])
-    assert np.var(mean_rewards) > 0
-    assert np.var(goals) > 0
+    assert np.var(mean_rewards) > 1e-3
```

Comment:
Doesn't this make the test more likely to fail? It should still be fine, I guess.

Reply:
I ran it for 100 times and the lowest variance is greater than 1e-2, so I think this should be fine.